ow_scholar/ow_scholar/requirements.txt

```diff
@@ -0,0 +1,47 @@
+aiodns==2.0.0
```

Comment:
Is this file automatically generated? I think we should freeze the dependencies for the Docker image, but I'd like to make freezing part of the Docker build so the developers can make whatever other version constraints they want.